.github/workflows/checks.yaml

```
@@ -57,7 +57,8 @@ jobs:
         uses: golangci/golangci-lint-action@v3.7.0
         with:
           version: v1.55.2
-          args: --timeout 10m0s
+          args: --timeout=10m0s --go=${{ env.golang-version }}
+          skip-pkg-cache: true
```

Can you add a comment
```suggestion
          # Disable caching as a workaround for https://github.com/golangci/golangci-lint-action/issues/135.
          # The line can be removed once the golangci-lint issue is resolved.
          skip-pkg-cache: true
```

---

Ideally, we don't want to skip the cache, but a fix upstream :P 

The comment will be a good reminder that we want to eventually enable cache again

---

Got it

---

Got it